sdk/keyvault/keyvault-keys/src/cryptographyClient.ts

```diff
@@ -65,12 +65,12 @@ export class CryptographyClient {
    *
    * Example usage:
    * ```ts
-   * let client = new CryptographyClient(url, key, credentials);
+   * let client = new CryptographyClient(keyVaultKey, credentials);
    * let result = await client.encrypt("RSA1_5", Buffer.from("My Message"));
    * ```
-   * @param algorithm The algorithm to use
-   * @param plaintext The text to encrypt
-   * @param options Additional options
+   * @param {JsonWebKeyEncryptionAlgorithm} algorithm The algorithm to use.
```

Comment:
```suggestion
   * @param {EncryptionAlgorithm} algorithm The algorithm to use.
```

Reply:
There are other occurrences like these in this file. Please correct those as well